### PR TITLE
feat: extra APIs

### PR DESCRIPTION
**Pull request by dongwoo-kim**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
- startEditing
- startEditingAt
- getColumns
- getColumnValues
- getIndexOfColumn
- setColumnValues
- activateFocus

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/grid.tsx [hunk 1]
```diff
@@ -341,6 +438,35 @@ export default class Grid {
...
+  public getColumns() {
+    return this.store.column.allColumns
+      .filter(({ name }) => !isRowHeader(name))
+      .map((column) => getOriginObject(column as Observable<ColumnInfo>));
+  }
+
+  /**
+   * Returns a list of all values in the specified column.
+   * @param {string} columnName - The name of the column
+   * @param {boolean} [isJsonString=false] - It set to true, return value will be converted to JSON string.
```
Comment (seonim-ryu) [inline src/grid.tsx]: 이 옵션 가져가는건가요??

Comment (dongwoo-kim) [reply src/grid.tsx]: 아.. 버려야죠 ㅋ

File: src/grid.tsx [hunk 2]
```diff
@@ -523,8 +649,7 @@ export default class Grid {
     this.dispatch('appendRow', row, options);
 
     if (options.focus) {
-      const rowIdx =
-        typeof options.at !== 'undefined' ? options.at : this.store.data.rawData.length - 1;
+      const rowIdx = isUndefined(options.at) ? this.getRowCount() - 1 : options.at;
```
Comment (seonim-ryu) [inline src/grid.tsx]: `rowIdx`인 이유는.. `rowIndex`가 길어서인가요?ㅎ`

File: src/grid.tsx [hunk 3]
```diff
@@ -264,16 +324,43 @@ export default class Grid {
...
+    this.dispatch('setNavigating', true);
+  }
 
-    if (typeof rowKey !== 'undefined' && name) {
-      result = this.focus(rowKey, name, isScrollable);
+  /**
+   * Sets focus on the cell at the specified index of row and column and starts to edit.
+   * @param {number|string} rowKey - The unique key of the row
+   * @param {string} columnName - The name of the column
+   * @param {boolean} [isScrollable=false] - If set to true, the view will scroll to the cell element.
```
Comment (seonim-ryu) [inline src/grid.tsx]: 이것도 빠지는건가요??

File: src/grid.tsx [hunk 4]
```diff
@@ -264,16 +324,43 @@ export default class Grid {
...
+      this.dispatch('startEditing', rowKey, columnName);
     }
+  }
 
-    return result;
+  /**
+   * Sets focus on the cell at the specified index of row and column and starts to edit.
+   * @param {number|string} rowIndex - The index of the row
+   * @param {string} columnIndex - The index of the column
+   * @param {boolean} [isScrollable=false] - If set to true, the view will scroll to the cell element.
```
Comment (seonim-ryu) [inline src/grid.tsx]: 여기도....

**Comment by seonim-ryu** [comment]
[5/30] api 추가군요ㅎ 고생하셨습니다.

File: src/grid.tsx (hunk 3 above)
Comment (dongwoo-kim) [reply src/grid.tsx]: 살려야겠네요 ㅎ

File: src/grid.tsx (hunk 4 above)
Comment (dongwoo-kim) [reply src/grid.tsx]: 여기도.. ㅎ

File: src/grid.tsx (hunk 2 above)
Comment (dongwoo-kim) [reply src/grid.tsx]: 흠.. 다음에 논의하죠 ㅎㅎ

**Pull request merged**